backend/build.gradle

```diff
@@ -1,35 +1,24 @@
 plugins {
-	id 'org.springframework.boot' version '2.4.5'
+	id 'org.springframework.boot' version '2.5.2'
 	id 'io.spring.dependency-management' version '1.0.11.RELEASE'
 	id 'java'
+	id 'war'
 }
 
 group = 'br.com.aegro'
-version = '1.0.0'
+version = '0.0.1'
 sourceCompatibility = '16'
 
 repositories {
 	mavenCentral()
 }
 
-ext {
-	set('testcontainersVersion', "1.15.3")
-	set('jackson.version', '2.12.3')
```

Comment:
If we want to support de/serialization of Java record classes, this line must be declared in the build.gradle.